README.md

```diff
@@ -36,24 +36,24 @@ Logging from the following services is supported for both cases:
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| alb\_logs\_prefix | S3 prefix for ALB logs. | string | `alb` | no |
-| allow\_alb | Allow ALB service to log to bucket. | string | `false` | no |
-| allow\_cloudtrail | Allow Cloudtrail service to log to bucket. | string | `false` | no |
-| allow\_cloudwatch | Allow Cloudwatch service to log to bucket. | string | `false` | no |
-| allow\_config | Allow Config service to log to bucket. | string | `false` | no |
-| allow\_elb | Allow ELB service to log to bucket. | string | `false` | no |
-| allow\_redshift | Allow Redshift service to log to bucket. | string | `false` | no |
-| allow\_s3 | Allow S3 service to log to bucket. | string | `false` | no |
-| cloudtrail\_cloudwatch\_logs\_group | The name of the CloudWatch Logs group to send CloudTrail events. | string | `cloudtrail-events` | no |
-| cloudtrail\_logs\_prefix | S3 prefix for CloudTrail logs. | string | `cloudtrail` | no |
-| cloudwatch\_logs\_prefix | S3 prefix for CloudWatch log exports. | string | `cloudwatch` | no |
-| config\_logs\_prefix | S3 prefix for AWS Config logs. | string | `config` | no |
-| default\_allow | Whether all services should be allowed by default. Individual services can override this default. | string | `true` | no |
-| elb\_logs\_prefix | S3 prefix for ELB logs. | string | `elb` | no |
-| redshift\_logs\_prefix | S3 prefix for RedShift logs. | string | `redshift` | no |
-| region | Region where the AWS S3 bucket will be created. | string | - | yes |
-| s3\_bucket\_name | S3 bucket to store AWS logs in. | string | - | yes |
-| s3\_log\_bucket\_retention | Number of days to keep AWS logs around. | string | `90` | no |
+| alb\_logs\_prefix | S3 prefix for ALB logs. | string | `"alb"` | no |
```

Comment:
this change looks related to pre-commit rather than updating circleCI image

Reply:
Surprisingly this actually belongs in this PR. In PR #21 the version of terraform-docs from the docker image is older (v0.5.0 I think) and so the terraform docs were correct still even though I updated the pre-commit hook versions. That's because the pre-commit hook version doesn't specify the terraform-docs version. But in this docker image the terraform-docs version is 0.6.0 and so the tests won't pass unless I also update these docs.  

It seems weird but its supposed to work this way :)